### PR TITLE
(DOCSP-39188): Clarify that restores can be at most one major version ahead

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
## DESCRIPTION

Clarify that target clusters for restores can be at most one major version ahead of the source cluster.

## STAGING

- [mongorestore](https://preview-mongodbjeffallenmongo.gatsbyjs.io/database-tools/DOCSP-39188/mongorestore/)
- [mongodump](https://preview-mongodbjeffallenmongo.gatsbyjs.io/database-tools/DOCSP-39188/mongodump/#definition)
- [mongorestore Behavior](https://preview-mongodbjeffallenmongo.gatsbyjs.io/database-tools/DOCSP-39188/mongorestore/mongorestore-behavior-access-usage/#restore-to-matching-server-version)
- [mongodump Behavior](https://preview-mongodbjeffallenmongo.gatsbyjs.io/database-tools/DOCSP-39188/mongodump/mongodump-behavior/#restore-to-matching-server-version)

## JIRA

https://jira.mongodb.org/browse/DOCSP-39188

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66354744b67dbf804ac09b70

(not sure what these errors are, but I'm fairly confident this PR did not introduce them)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jmd-mongo — APPROVED** [review]
Thanks for these updates, @jeff-allen-mongo! This one LGTM.

Thanks,
Joe

**Review by mvankeulen94 — CHANGES_REQUESTED** [review]
Apologies for the confusion. The issue I'd like to solve for is to make sure that users know where our recommended upgrade procedures are. The BSON claims currently being made aren't wrong, but we don't want users to be tempted to try to use dump and restore as a way to circumvent our standard upgrade procedures. Thanks!

**Review by jeff-allen-mongo — COMMENTED** [review]
@mvankeulen94 thanks for the feedback! I adjusted the copy to focus more on clarifying that dump / restore should not be used to upgrade a deployment. Could you take another look?

File: source/includes/mongodump-restore-version-link.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+You can restore the BSON files generated from ``mongodump`` into MongoDB
+deployments running the same or newer version as the source deployment.
+However, restoring files into a newer version deployment is not a
+suitable way to upgrade your deployment. To learn how to upgrade your
```
Comment (mvankeulen94) [inline source/includes/mongodump-restore-version-link.rst:4]: "a suitable" -> "the recommended"

**Review by mvankeulen94 — CHANGES_REQUESTED** [review]
Just a couple minor wording changes, thanks!
I'll also flag this to TAR engineering & TS so they're aware that these changes are being made.

**Comment by jeff-allen-mongo** [comment]
Thanks @mvankeulen94! Made the suggested edits.

**Review by mvankeulen94 — APPROVED** [review]
looks great, thank you!

**Pull request merged**